test/integration/targets/azure_rm_batchaccount/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+---
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#
+# ----------------------------------------------------------------------------
+- name: Prepare random number
+  set_fact:
+    storage_account_name: "st{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+    batch_account_name: "ba{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+  run_once: yes
+
+- name: Create Storage Account
```

Comment:
We should delete this, people do run these manually sometimes, and we don't want to leave storage accounts hanging